modules/eks/cert-manager/main.tf

```diff
@@ -125,15 +118,15 @@ module "cert_manager_issuer" {
   repository           = var.cert_manager_issuer_repository
   description          = var.cert_manager_issuer_description
   chart_version        = var.cert_manager_issuer_chart_version
-  kubernetes_namespace = join("", kubernetes_namespace.default.*.id)
+  kubernetes_namespace = var.kubernetes_namespace
   create_namespace     = false
   wait                 = var.wait
   atomic               = var.atomic
   cleanup_on_fail      = var.cleanup_on_fail
   timeout              = var.timeout
 
   # Only install IAM role if letsencrypt_enabled is true
```

Comment:
remove comment if no longer an option
```suggestion
```

Reply:
@milldr I do not understand your comment. The code comment is still accurate.